sql-plugin/pom.xml

```diff
@@ -393,6 +393,32 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
+                <artifactId>scala-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.scalatest</groupId>
+                <artifactId>scalatest-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
There's already an antrun plugin section in this pom, so this should be collected there to reduce boilerplate and keep the usage of a particular plugin centralized.

Reply:
merged with the one above it